src/v2/views/ErrorView/index.tsx

```diff
@@ -133,8 +135,14 @@ function ErrorView() {
         </ErrorContent>
       </Route>
       <Route path={`${path}/reinstall`}>
-        <ErrorContent title={t("Reinstall required", { _tags: transifexTags })}>
+        <ErrorContent
+          title={t("Reinstall required edit edit", { _tags: transifexTags })}
+        >
           <T _str="Please reinstall Nine Chronicles." _tags={transifexTags} />
+          <T
```

Comment:
You can navigate the user to the browser via: https://www.electronjs.org/docs/latest/api/shell#shellopenexternalurl-options

Reply:
Sorry for i late
I'll try the way you told me.
thx!! 🤟🏻